.env.example

```diff
@@ -14,7 +14,7 @@ LAKEHOUSE_ABFSS_PREFIX=
 TEST_CONFIG_PATH=
 EXTRACTOR_EVENT_PATH=
 EXTRACTOR_FILE_PATH=
-EXTRAXTOR_RAW_TS_PATH=
+EXTRACTOR_RAW_TS_PATH=
```

Comment:
I know its a small change, but to keep this focused  i would recommend doing this change in a separate PR and keep this one just about documentation

Reply:
I'd argue it's in scope, this is a documentation change rather than a code change and since I explain the evironment variables in the documentation I'd prefer to fix the typo now

Reply:
The typo was fixed in the main branch